x/feemarket/client/cli/query.go

```diff
@@ -36,27 +30,20 @@ func GetQueryCmd() *cobra.Command {
 // GetBlockGasCmd queries the gas used in a block
 func GetBlockGasCmd() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "block-gas [height]",
+		Use:   "block-gas",
 		Short: "Get the block gas used at a given block height",
 		Long: `Get the block gas used at a given block height.
 If the height is not provided, it will use the latest height from context`,
-		Args: cobra.RangeArgs(0, 1),
+		Args: cobra.NoArgs,
```

Comment:
these changes are unrelated to the PR, please split it into a separate PR

Reply:
done, extracted to https://github.com/evmos/ethermint/pull/1230